apps/sms/js/thread_ui.js

```diff
@@ -734,10 +736,10 @@ var ThreadUI = {
       // Recipients will exist for draft messages in threads
       // Otherwise find them from draft recipient numbers
       draft.recipients.forEach(function(number) {
-        Contacts.findByAddress(number, function(records) {
-          if (records.length) {
+        Contacts.findByAddress(number).then(function(contacts) {
+          if (contacts.length) {
```

Comment:
theorically but the cases where we get "null" are really specific; I don't think this can really happen here. That said I agree we should just get rid of this possibility.